tissot: Update 3.13

answer35 opened the pull request.

Commit:
tissot: Initial release changelogs

Commit:
tissot: Update 3.13

Commit:
tissot: Latest gapps update

Commit:
tissot: Latest vanilla update

Commit:
tissot: copied json files to web folder

rajkale99 reviewed (changes requested):
For now don't add json in Tissot/official/version.json, 
Only keep web folder one this time

answer35 commented:
It actually has been automatic with script :/ I will remove it then

Commit:
tissot: removed generated json files

Removed generated json files that were in official folder after copied them into official/web folder instead

answer35 commented:
Files has been removed.

Commit:
Update json file location to official/web folder

rajkale99 reviewed (approved):
Nice
